Makefile

```diff
@@ -76,15 +76,13 @@ $(GOIMPORTS):
 build: format
 	go build -v ./...
 	cp wavefront/config/wavefront.yaml install/wavefront/templates/
-	sed -i '' -E 's/namespace: istio-system/namespace: {{ .Values.namespaces.istio }}/' install/wavefront/templates/wavefront.yaml
-	sed -i '' -E 's/- metric/- metric.{{ .Values.namespaces.adapter }}/' install/wavefront/templates/wavefront.yaml
```

Comment:
Why does this need to go?

Reply:
i guess helm is taking care to replace these so do we still need them?  And also sed is failing in ubuntu and redhat machines.

Reply:
Helm doesn’t replace it, so we need these commands. Can you try adding a operating system check for macOs v/s others and run the correct sed command for linux systems? The current one only works with mac. I think dropping -E will be sufficient to make it work on Linux.

Reply:
I don't seem to find a way to make this replacement using sed in linux, i guess the issue is with special characters `{{` and `.` tried with escaping them still sed didn't work. If they are not getting replaced by helm, can those be hard coded?

Reply:
Reverting as adapter can be built on mac.